pyproject.toml

```diff
@@ -56,6 +56,10 @@ pytest = "^7.1.3"
 pytest-cov = "^4.0.0"
 pytest-sugar = "^0.9.5"
 
+
+[tool.poetry.group.dev.dependencies]
+pydocstyle = "^6.1.1"
```

Comment:
We use `flake8-docstrings` which uses pydocstyle as a flake8 plugin. We shouldn't need this dependency